build.js

```diff
@@ -61,9 +61,9 @@ if (shell.exec(`rollup -c rollup.es.config.js -i ${NPM_DIR}/${PACKAGE}.js -o ${F
 }
 
 shell.echo(`Produce ESM5/FESM5 versions`);
-shell.exec(`ngc -p ${OUT_DIR}/tsconfig-build.json --target es5 -d false --outDir ${OUT_DIR_ESM5} --sourceMap`);
+shell.exec(`tsc -p ${OUT_DIR}/tsconfig-build.json --target es5 -d false --outDir ${OUT_DIR_ESM5} --sourceMap`);
```

Comment:
Move to typescript compiler is not correct. The error is in tsickle ^0.33.0 https://github.com/angular/tsickle/issues/930

Reply:
I didn't know what is the exact cause of that error, you know better 👍